graphql-dgs-codegen-core/src/main/kotlin/com/netflix/graphql/dgs/codegen/generators/kotlin/KotlinDataTypeGenerator.kt

```diff
@@ -171,8 +171,8 @@ abstract class AbstractKotlinDataTypeGenerator(private val packageName: String,
     }
 
     private fun addToString(fields: List<Field>, kotlinType: TypeSpec.Builder): String {
-        val toStringBody = StringBuilder("return \"{\" + ")
-        fields.mapIndexed { index, field ->
+        val toStringBody = StringBuilder("return linkedMapOf(\n")
+        fields.map { field ->
```

Comment:
When I tried this out with a nullable Int type, I don't see a null check added for the Integer field. Am I missing something here?